build.gradle

```diff
@@ -102,13 +106,68 @@ dependencies {
 
     implementation "org.apache.commons:commons-text:1.9"
 
+    compileOnly "io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion"
```

Comment:
Being `compileOnly` means it won't be registered as the dependency of the package, i.e. won't be transitively used by the connector libraries. Is it intentional?

Reply:
Yes I use `AvroData` for converting connect schema to avro schema